docker-compose.yml

```diff
@@ -1,42 +1,51 @@
-version: '2'
+version: '3'
 services:
     database:
-        container_name: database
         environment:
             MYSQL_ROOT_PASSWORD: root
             MYSQL_DATABASE: symfony
             MYSQL_USER: symfony
             MYSQL_PASSWORD: symfony
         image: mariadb:10
         networks:
-            - default
+            - backend
         ports:
             - "3306:3306"
         volumes:
             - ./database:/var/lib/mysql
     php:
         build: ./php-fpm
-        container_name: php
+        depends_on:
+            - database
         links:
             - database
         networks:
-            - default
+            - backend
+            - frontend
+        expose: 
+            - "9000"
         volumes:
-            - ./symfony:/var/www/symfony
-            - ./logs/symfony:/var/www/symfony/var/log
+            - ./symfony:/var/www
+            - ./logs/symfony:/var/www/var/log
+        environment:
+            - COMPOSER_MEMORY_LIMIT=-1
     nginx:
         build: ./nginx
-        container_name: nginx
```

Comment:
why remove all the container_name ? It's pretty usefull if you wants to manipulate your containers outside of the project folder. And `docker-symfony_nginx_1` is pretty long to write !